modules/core/04-channel/keeper/keeper_test.go

```diff
@@ -484,6 +501,28 @@ func (suite *KeeperTestSuite) TestSetRestoreChannel() {
 	suite.Require().Equal(expChannel, restoreChannel)
 }
 
+func (suite *KeeperTestSuite) TestDeleteUpgradeRestoreChannel() {
```

Comment:
nit: But I think you can test all store functions, `Set/Get/Delete` in a single test function. The prior test is just a subset of this test

Reply:
Agree!

Reply:
I refactored the tests to use subtests for each field 👍